src/Model.h

```diff
@@ -90,6 +90,17 @@ struct ContactEvent
 	unsigned short int contact_time;
 };
 
+/**
+ * @brief Apply place closure effects to household in a clearer way, to avoid *the appearance* of a data race.
+ *
+ */
```

Comment:
There is actually a real race that can happen: If you have one thread doing
```
if (absent_start_time > 5)
    absent_start_time = 5;
```
and another doing
```
if (absent_start_time > 7)
    absent_start_time = 7;
```
at the same time with `absent_start_time` initially 9 then these could run as
```
if (absent_start_time > 7)
if (absent_start_time > 5)
    absent_start_time = 5;
    absent_start_time = 7;
```
leaving it set to 7, rather than 5. So I'd suggest changing the comment to `Apply place closure effects to household in a thread-safe way.`

Reply:
Yes, good point - comment changed.